e2e/cypress/e2e/wire/wire_action.cy.js

```diff
@@ -59,48 +31,46 @@ describe('wire - filters', () => {
         /**
             SEARCH
         */
-        cy.get('[data-test-id="sidenav-link-wire"]').click();
-        cy.get('[data-test-id="top-search-bar"]').click().type('Sydney');
-        cy.get('[data-test-id="search-submit-button"]').click();
+        WirePage.openSideNav();
+        WirePage.search('Sydney');
 
         cy.url().should('include', 'Sydney');
 
-        cy.get(`[data-test-id="wire-item"][data-test-value="${WIRE_ITEMS.syd_weather_1._id}"]`).should('exist');
-        cy.get(`[data-test-id="wire-item"][data-test-value="${WIRE_ITEMS.bris_traffic_1._id}"]`).should('not.exist');
+        cy.get(`[data-test-id="wire-item"][data-test-value="${WireItems.item_1._id}"]`).should('exist');
```

Comment:
you could also make a helper for this like `WirePage.item(WireItems.item_1).should('exist');`